sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.aest.controller.ts

```diff
@@ -69,4 +72,23 @@ export class ApplicationOfferingChangeRequestAESTController extends BaseControll
       count: offeringChange.count,
     };
   }
+
+  /**
+   * Gets the Application Offering Change Request details.
+   * @param applicationOfferingChangeRequestId the Application Offering Change Request id.
+   * @returns Application Offering Change Request details.
+   */
+  @Get(":applicationOfferingChangeRequestId")
+  @ApiNotFoundResponse({
+    description: "Not able to find an Application Offering Change Request.",
+  })
+  async getApplicationOfferingChangeRequest(
+    @Param("applicationOfferingChangeRequestId", ParseIntPipe)
+    applicationOfferingChangeRequestId: number,
+  ): Promise<ApplicationOfferingDetailsReviewAPIOutDTO> {
```

Comment:
Why is the DTO named with review? looks like it is influenced by UI.

Reply:
It is the returning the properties for the ministry view. That is why I named it such.

Reply:
But still it is not matching the purpose IMO. The purpose of DTO is to have all the details of application offering change.

Reply:
@dheepak-aot I am not getting a good name for this DTO. Can you please suggest some possible options.

Reply:
ApplicationOfferingChangeDetailsAPIOutDTO 😊

I do not see this name currently anywhere.